src/components/governance/ProposalDetails.tsx

```diff
@@ -260,7 +260,7 @@ function ProposalDetails({
             </AutoColumn>
             <AutoColumn gap="md">
               <MarkDownWrapper>
-                <ReactMarkdown source={proposalData?.description} />
+                <ReactMarkdown source={proposalData?.description} disallowedTypes={["code"]} />
```

Comment:
Theres still some weird styling stuff going on with overflows in the description area.